src/main/groovy/edu/oregonstate/mist/locations/frontend/LocationConfiguration.groovy

```diff
@@ -0,0 +1,27 @@
+package edu.oregonstate.mist.locations.frontend
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import edu.oregonstate.mist.api.Credentials
+import io.dropwizard.Configuration
+import io.dropwizard.client.JerseyClientConfiguration
+
+import javax.validation.Valid
+import javax.validation.constraints.NotNull
+
+class LocationConfiguration extends Configuration {
```

Comment:
consider the cosmetic change to rename this class to `LocationsFrontendApplicationConfiguration`, `LocationsFrontendConfiguration`, or `LocationsConfiguration` for consistency. maintainers may not remember which location classes are plural and which are singular.